### ISSUE TITLE
Quick Action: POI Overlay is missing the search field

### ISSUE THREAD

**Issue by fritzphantom** [osmandapp/OsmAnd#19908]
### Description

When trying to define a quick action to enable certain POI, only the main categories are shown and can be selected. The search field to look for more nuanced POIs is missing. It is there however when doing the same over: configure map->enable poi->and there is then search field.

### Steps to reproduce

Go to configure maps-> enable POI -> there is the search field, try to search for e.g.: "charging station"

Try adding a quick action->enable POI-> there this search field is missing, and only the major categories can be selected.

### Actual result

The search field is not there.

### Expected result

The search field should be there.

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version:
Android/iOS version:
Device model:
Crash-Logs: ?
```

**Comment by zipav** [comment]
OsmAnd Nightly 4.8.0#2380m, released: 2024-05-24

Not reproduced. 


https://github.com/osmandapp/OsmAnd/assets/123936521/78a3e8a5-faf2-4f3b-82e9-15978a3fe7cd

**Comment by fritzphantom** [comment]
![grafik](https://github.com/osmandapp/OsmAnd/assets/61949537/05c8e2a8-fa8e-43a2-bb61-83fed57e6693)

I would expect here a search field, just as it is when going to:
configure maps-> enable POI -> there is the search field

Do you see it there?

**Comment by Guylby** [comment]
I agree that the Search field which is present on the POI overlay menu would be really helpful on the Custom button action menu!

I find custom buttons really useful to prevent clutter on the map, with one button showing only Fuel station, another just Parkings for example, but as of know we have no granularity control as it seems not possible to create a custom POI filter with the custom buttons menu nor search for certain types of POI.

Using the Enable POI menu is nice but it will be overriden once you press one of your customs buttons (previous POI will be hidden and the ones set with the button shown), defeating the purpose as you would need to go to the configure map again and set it up.

**Comment by symphorien** [comment]
as a workaround you can add your own category to the main categories listed when adding a quick action:
- click search
- select the categories tab
- type bakery
- instead of clicking "show charging station on the map", click on the funnel icon on the left
- click on the three dots menu, then save filter
- accept the name, save

now when adding a POI category to the quick action button, bakery is listed:

<img width="320" alt="Image" src="https://github.com/user-attachments/assets/bfcda708-1474-4c33-be4f-1cfac7734564" />

EDIT: initially I gave this example with one of the default categories which is not very useful
